src/_includes/soft_mod_hubspot_in_page_banner_cta.liquid

```diff
@@ -0,0 +1,19 @@
+{%- if site.lang == 'es' -%}
+  <span class="hs-cta-wrapper hubspot-in-page-banner" id="hs-cta-wrapper-a4f30da8-3062-4c47-a865-a04a70bfa951">
+    <span class="hs-cta-node hs-cta-a4f30da8-3062-4c47-a865-a04a70bfa951" id="hs-cta-a4f30da8-3062-4c47-a865-a04a70bfa951">
+      <!--[if lte IE 8]><div id="hs-cta-ie-element"></div><![endif]-->
+      <a href="https://cta-redirect.hubspot.com/cta/redirect/3042464/a4f30da8-3062-4c47-a865-a04a70bfa951" ></a>
+    </span>
+    <script charset="utf-8" src="https://js.hscta.net/cta/current.js"></script>
+    <script type="text/javascript"> hbspt.cta.load(3042464, 'a4f30da8-3062-4c47-a865-a04a70bfa951', {}); </script>
+  </span>
+{%- else -%}
+  <span class="hs-cta-wrapper hubspot-in-page-banner" id="hs-cta-wrapper-39e3f247-bc43-4085-a52b-73a8d0930da9">
+    <span class="hs-cta-node hs-cta-39e3f247-bc43-4085-a52b-73a8d0930da9" id="hs-cta-39e3f247-bc43-4085-a52b-73a8d0930da9">
+      <!--[if lte IE 8]><div id="hs-cta-ie-element"></div><![endif]-->
+      <a href="https://cta-redirect.hubspot.com/cta/redirect/3042464/39e3f247-bc43-4085-a52b-73a8d0930da9" ></a>
+    </span>
+    <script charset="utf-8" src="https://js.hscta.net/cta/current.js"></script>
+    <script type="text/javascript"> hbspt.cta.load(3042464, '39e3f247-bc43-4085-a52b-73a8d0930da9', {}); </script>
+  </span>
+{%- endif -%}
```

Comment:
There's masses of repetition between both sides of the conditional and also between the two hubspot_cta includes. So we have repetition in four places.

It looks like the only variables are the Hubspot ID and the class name we are using. Should we have a single include and just pass these two things in as parameters?

Reply:
That's a good idea 👍